### PR TITLE
✨ Source Salesforce, Shopify: add maxSecondsBetweenMessages in metadata

### PR DESCRIPTION
**Pull request by strosek**
Certified connectors are required to have `maxSecondsBetweenMessages` set in metadata file.
`maxSecondsBetweenMessages` is the longest time frame (in seconds) of API requests limits reset for endpoints used by connector. [Issue](https://app.zenhub.com/workspaces/critical-connectors-659c693d906fcd00178900cf/issues/gh/airbytehq/airbyte-internal-issues/6779)

## What
`maxSecondsBetweenMessages` value for the following connectors was added:

source-salesforce [reference](https://developer.salesforce.com/docs/atlas.en-us.salesforce_app_limits_cheatsheet.meta/salesforce_app_limits_cheatsheet/salesforce_app_limits_platform_api.htm)
source-shopify [reference](https://shopify.dev/docs/api/usage/rate-limits)

## How
Files `metadata.yaml` for connectors mentioned above were updated by adding `maxSecondsBetweenMessages` value.

## 🚨 User Impact 🚨
No breaking changes

### CODE REVIEW

File: airbyte-integrations/connectors/source-shopify/metadata.yaml
Comment (maxi297) [inline airbyte-integrations/connectors/source-shopify/metadata.yaml:20]: @bazarnov does it make sense in terms of rate limit for shopify (i.e. the rate limit window is one second)? @davinchia mentioned that there would still be a minimum. @davinchia, can you confirm the behavior when the `maxSecondsBetweenMessages` is lower than 90 minutes?

Comment (bazarnov) [reply airbyte-integrations/connectors/source-shopify/metadata.yaml:20]: @maxi297 It doesn't regard to the BULK Jobs running in avg 150 - 600 sec. I would rather reference the Shopify Documentation, where this info could have been, but there is no mentioned of how long the regular bulk job would take to run. My opinion here is we should rely on the value of 2 hours, and make it possible to adjust the slice size from the source itself, so the source logic follows the timings as well, when it's possible (if we have long-running jobs - we decrease the next slice to try-run faster, and if we have faster runs for bulk jobs, we can increase the slice size to try get bigger data portion next time, thus dynamically balance the slice size - working on it, btw)

so, @strosek it should be at least: `7200` for `source-shopify` and `86400` for `source-salesforce`, since the max bulk job time for `salesforce` is up to 1 day.

Comment (maxi297) [reply airbyte-integrations/connectors/source-shopify/metadata.yaml:20]: So, we've been told by @davinchia  that this value should be the value of the rate limit hence why I asked him to confirm that. Given the naming of the variable, your reasoning sounds good but this was not the ask given to us. @davinchia, please confirm

Comment (maxi297) [reply airbyte-integrations/connectors/source-shopify/metadata.yaml:20]: @davinchia confirmed @bazarnov point. Can we update shopify value? I'll approve so that once the change is done, you can merge.

**Review by maxi297 — APPROVED** [review]
Conditional approval on the resolution of the comment

**Pull request merged**
